panoramapublic/src/org/labkey/panoramapublic/message/PrivateDataMessageScheduler.java

```diff
@@ -0,0 +1,126 @@
+
+            // Clear previous job, if present
+            if (scheduler.checkExists(TRIGGER_KEY))
+                scheduler.unscheduleJob(TRIGGER_KEY);
+
+            if (!enable)
+            {
+                return;
+            }
+
+            // Get the quartz Trigger
+            Trigger trigger = getTrigger();
+
+            // Create a quartz job that queues a pipeline job that posts private data reminder messages
+            JobDetail job = JobBuilder.newJob(PrivateDataMessageSchedulerJob.class)
+                    .withIdentity(PrivateDataMessageScheduler.class.getCanonicalName())
+                    .build();
+
+            // Schedule trigger to send reminders on the configured schedule
+            scheduler.scheduleJob(job, trigger);
+        }
+        catch (SchedulerException e)
+        {
+            throw new RuntimeException("Failed to schedule PrivateDataMessageScheduler job", e);
+        }
+    }
+
+    protected Trigger getTrigger()
+    {
+        // Run at 8:00AM every morning
+        return TriggerBuilder.newTrigger()
+                .withIdentity(TRIGGER_KEY)
+                .withSchedule(CronScheduleBuilder.dailyAtHourAndMinute(8, 0))
+                .build();
+    }
+
+    public static class PrivateDataMessageSchedulerJob implements Job
+    {
+        @SuppressWarnings("unused")
+        public PrivateDataMessageSchedulerJob() {}
+
+        @Override
+        public void execute(JobExecutionContext context)
+        {
+            try
+            {
+                Journal panoramaPublic = JournalManager.getJournal(JournalManager.PANORAMA_PUBLIC);
+                if (panoramaPublic == null)
+                {
+                    throw new ConfigurationException("Server does not have a Panorama Public project.");
+                }
+
+                Container c = panoramaPublic.getProject();
+
+                User panoramaPublicAdmin = JournalManager.getJournalAdminUser(panoramaPublic);
+                if (panoramaPublicAdmin == null)
+                {
+                    throw new ConfigurationException("Unable to find an admin user in the Panorama Public project.");
+                }
+                ViewBackgroundInfo vbi = new ViewBackgroundInfo(c, panoramaPublicAdmin, null);
+                PipeRoot root = PipelineService.get().findPipelineRoot(c);
+
+                if (root == null || !root.isValid())
+                {
+                    throw new ConfigurationException("No valid pipeline root found in the container " + c.getName());
+                }
+
+                PipelineJob job = new PrivateDataReminderJob(vbi, PipelineService.get().getPipelineRootSetting(c), false);
+                PipelineService.get().queueJob(job);
+            }
+            catch(Exception e)
+            {
+                _log.error("Error queuing PrivateDataReminderJob", e);
+                ExceptionUtil.logExceptionToMothership(null, e);
```

Comment:
You're already logging the exception to the local log file, so no need for ExceptionUtil to log it again.

```suggestion
                ExceptionUtil.logExceptionToMothership(null, e, false);
```